src/components/cards/AssetStatusCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { AssetStatus } from 'edge-info-server/types'
+import * as React from 'react'
+import { Platform } from 'react-native'
+import { getLocales } from 'react-native-localize'
+import IonIcon from 'react-native-vector-icons/Ionicons'
+
+import { openBrowserUri } from '../../util/WebUtils'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { StatusCard } from './StatusCard'
+
+export const AssetStatusCard = (props: { assetStatus: AssetStatus }) => {
+  const { statusType, localeStatusTitle, localeStatusBody, iconUrl, statusUrl, statusStartIsoDate, statusEndIsoDate } = props.assetStatus
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const isWarning = statusType === 'warning'
+  const curDate = new Date().toISOString()
+  const isWithinDate = statusStartIsoDate != null && statusEndIsoDate != null && statusStartIsoDate <= curDate && statusEndIsoDate >= curDate
+
+  const [firstLocale = { languageTag: 'en_US' }] = getLocales()
+  const { languageTag } = firstLocale
+  const title = localeStatusTitle[languageTag]
+  const message = localeStatusBody[languageTag]
+
+  return isWithinDate ? (
+    <StatusCard
+      message={message}
+      title={title}
+      iconOrUri={
+        // If not explicitly set, auto-fill if warning status.
+        iconUrl == null ? (
+          isWarning ? (
+            <IonIcon
+              name={Platform.OS === 'ios' ? 'ios-warning-outline' : 'md-warning-outline'}
+              style={styles.icon}
+              color={theme.warningText}
+              size={theme.rem(3)}
+            />
+          ) : undefined
```

Comment:
For `statusType === 'info'` we should use some type of "i" icon if the user didn't give an iconUrl